.bumpversion.cfg

```diff
@@ -3,7 +3,7 @@ current_version = 0.5.0
 commit = True
 message = [skip ci] docs: Update version numbers from {current_version} -> {new_version}
 
-[bumpversion:file:core/version.go]
+[bumpversion:file:common/version.go]
```

Comment:
Note: I needed the current version in the new common package so I copied version.go there, plus we'll be moving the core package elsewhere so this seemed like a good idea :)